js/calc.js

```diff
@@ -0,0 +1,115 @@
+document.addEventListener("DOMContentLoaded",function(event){
+ 
+	
+    // global variables for calculator
+	var number1 = true;
+	var number2 = false;
+	var ops = false;
+
+	// setting the variables
+
+	function SetElement(){
+
+		var numbers = document.getElementsByClassName('buttonNum');
+		var operate = document.getElementsByClassName('operator');
+		var clear = document.getElementsByClassName('buttonClear');
+		var equal = document.getElementsByClassName('equals');
+
+		ButtonClick(numbers);
+		OperatorClick(operate);
+		clear[0].addEventListener('click', ClearCalc);
+		equal[0].addEventListener('click', Result);
+
+
+	}
+
+	// Resets the calculator
+
+	function ClearCalc(){
+		var screen = document.getElementById('screen');
+		screen.innerHTML = "Sparta Calculator"
+	    
```

Comment:
You also need to clear all your variables storing the number and operator data, otherwise you can't perform any operations after the screen has been cleared.